google-account-plugin/resources/META-INF/plugin.xml

```diff
@@ -57,6 +57,8 @@
                         serviceImplementation="com.google.cloud.tools.intellij.DefaultAccountPluginConfigurationService"/>
     <applicationService serviceInterface="com.google.cloud.tools.intellij.login.GoogleLoginService"
                         serviceImplementation="com.google.cloud.tools.intellij.login.IntellijGoogleLoginService"/>
+    <applicationService serviceInterface="com.google.cloud.tools.intellij.analytics.IdeaUsageTracker"
```

Comment:
You'll definitely need to do an Appu review.  A lot of the current
architecture is to support Android Studio tracking so we need to make sure
the new changes support those requirements.

On Fri, May 27, 2016 at 6:53 PM, Chanseok Oh <notifications@github.com>
wrote:

> In google-account-plugin/resources/META-INF/plugin.xml
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/699#discussion_r64934962>
> :
>
> > @@ -57,6 +57,8 @@
> >                          serviceImplementation="com.google.cloud.tools.intellij.DefaultAccountPluginConfigurationService"/>
> >      <applicationService serviceInterface="com.google.cloud.tools.intellij.login.GoogleLoginService"
> >                          serviceImplementation="com.google.cloud.tools.intellij.login.IntellijGoogleLoginService"/>
> > +    <applicationService serviceInterface="com.google.cloud.tools.intellij.analytics.IdeaUsageTracker"
>
> The plan is to do this in multiple steps to lessen the review burden. The
> first step is to introduce and set up new facilities. The second is to
> change exist code to use it. The last is to remove old code, which is
> pretty large and complex.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/699/files/bde422900095e05a7dcf1625850ae2a11fd65cfa#r64934962>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHf5HeW9kirlHDkPCGYIgfZCmYp2rHKHks5qFyGSgaJpZM4ImAuk>
> .
>